cdap-storage-spi/src/main/java/io/cdap/cdap/spi/data/StructuredTable.java

```diff
@@ -182,7 +182,7 @@ default CloseableIterator<StructuredRow> scan(Range keyRange, int limit, Field<?
    *                               or the type does not match the schema
    * @throws IOException if there is an error scanning the table
    */
-  default CloseableIterator<StructuredRow> scan(Range keyRange, int limit, String orderByField, SortOrder sortOrder)
+  default CloseableIterator<StructuredRow> scan(Range keyRange, int limit, Field<?> orderByField, SortOrder sortOrder)
```

Comment:
@de-lan This needs to be a Field for no-sql implementation

Reply:
got it, will update my PR accordingly

Reply:
Why does this need to be a Field? String seems correct, we just need the column name and not a value.

Reply:
@albertshau  For nosql, we also need the type. But second thought... @sanjanasandeep we could get the type out from schema

Reply:
@sanjanasandeep Did you rebase to `develop`? I remember I've updated this part as well. If you rebased, it should not shown in diff

Reply:
yes looks like we can get it from schema as well, maybe better to use column-name? I can change that in the sql implementation too @de-lan